website/docs/cloud-docs/api-docs/run-tasks/run-tasks-integration.mdx

```diff
@@ -36,11 +38,15 @@ The service receiving the run task request should respond with `200 OK`, or Terr
 
 The POST request submits a JSON object with the following properties as a request payload.
 
+#### Common Properties
+
+All request payloads contain the following properties.
+
 |            Key path             |  Type   |          Values          |                                                 Description                                                 |
 | ------------------------------- | ------- | ------------------------ | ----------------------------------------------------------------------------------------------------------- |
 | `payload_version`               | integer | `1`                      | Schema version of the payload. Only `1` is supported.                                                       |
 | `access_token`                  | string  |                          | Bearer token to use when calling back to Terraform Cloud.                                                   |
-| `stage`                         | string  | `post-plan`, `pre-apply` | The [stage](/cloud-docs/run/states) the task was triggered at.                                              |
+| `stage`                         | string  | `pre_plan`, `post_plan`, `pre_apply` | The [stage](/cloud-docs/run/states) the task was triggered at.                                              |
```

Comment:
You mentioned this is in beta right? I forgot that during my first pass! Also forgive me if I'm adding an incorrect actor again. I just hate ending on "at" - let me know if this works!

```suggestion
| `stage`                         | string  | `pre_plan`, `post_plan`, `pre_apply` | The [run stage](/cloud-docs/run/states) when Terraform Cloud triggers the run task.  The `stage` property is in beta.                                            |
```